jboss/container/wildfly/launch/datasources/test/datasource/configure.bats

```diff
@@ -308,6 +308,61 @@ load common
     assert_datasources "prefix_ibmdb2_xa.xml"
 }
 
+@test "inject_datasources: DB_SERVICE_PREFIX_MAPPING - DRIVER_CLASS" {
+    DB_SERVICE_PREFIX_MAPPING="test-postgresql=TEST"
+    TEST_JNDI="java:/jboss/datasources/testds"
+    TEST_DRIVER="postgresql"
+    TEST_USERNAME="kermit"
+    TEST_PASSWORD="thefrog"
+    TEST_URL="jdbc:postgresql://localhost:5432/postgresdb"
+    TEST_NONXA="true"
+    TEST_JTA="false"
+
+    TEST_DRIVER_CLASS="org.postgresql.Driver"
+
+    run inject_datasources
+
+    [ "$status" -eq 0 ]
+    assert_datasources "prefix_postgresql_nonxa_driver_class.xml"
+}
+
+
+@test "inject_datasources: DB_SERVICE_PREFIX_MAPPING - DATASOURCE_CLASS" {
+    DB_SERVICE_PREFIX_MAPPING="test-postgresql=TEST"
+    TEST_JNDI="java:/jboss/datasources/testds"
+    TEST_DRIVER="postgresql"
+    TEST_USERNAME="kermit"
+    TEST_PASSWORD="thefrog"
+    TEST_URL="jdbc:postgresql://localhost:5432/postgresdb"
+    TEST_NONXA="true"
+    TEST_JTA="false"
+
+    TEST_DATASOURCE_CLASS="org.postgresql.Driver"
```

Comment:
Nitpick: Would you mind using a real Postgres SQL data source instead? e.g. `org.postgresql.jdbc2.optional.SimpleDataSource`

It is just a test class and it doesn't matter, but, these tests are useful as documentation and copy&paste from them to test the images is a common task.